### PR TITLE
feat: additional token launcher components

### PR DESCRIPTION
**Pull request by ashmidgley**
[Wizard setup](https://www.notion.so/rational-nomads/Wizard-setup-e89ce4c065004ea09ee3d789db4b9341)

-----

## 1. Pull request checklist
- [x] Notion card has been moved to the Code Review column
- [x] Notion card has a link to this PR
- [x] A reviewer has been assigned to the Notion card

## 2. PR type
```
- Feature
```

## 3. What is the old behaviour?
Missing a few components and some behaviour in existing components required for the Token Launcher project.

## 4. What is the new behaviour?
Addition of the InfoTooltip, cross and question mark icons and tweaks to Wizard Header to allow rendering of children.

### CODE REVIEW

File: src/components/Icons/icons/IconCross.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import type { FC } from 'react';
```
Comment (colbr) [inline src/components/Icons/icons/IconCross.tsx:1]: Nice - thanks for adding these! Just FYI we're about to add a script which automatically creates React components from all Figma icons 🚀

File: src/components/InfoTooltip/InfoTooltip.scss [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  flex-direction: column;
+  justify-content: center;
+
+  .zui-info-tooltip-tile {
+    cursor: pointer;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    width: 16px;
+    height: 16px;
```
Comment (colbr) [inline src/components/InfoTooltip/InfoTooltip.scss:15]: Switch these to `1rem`.

File: src/components/MediaInput/MediaInput.scss [hunk 3]
```diff
@@ -29,7 +29,19 @@
...
+      flex-direction: column;
+      text-align: center;
+
+      .zui-media-input-preview-text-title {
+        font-size: 16px;
+        color: #{$text-glow-white};
+      }
+  
+      .zui-media-input-preview-text-subtitle {
+        font-size: 14px;
```
Comment (colbr) [inline src/components/MediaInput/MediaInput.scss:42]: Switch both `font-size`s to `rem` values.

File: src/components/MediaInput/index.ts [hunk 4]
```diff
@@ -1 +1,2 @@
 export * from './MediaInput';
+export * from "./MediaInput.types";
```
Comment (colbr) [inline src/components/MediaInput/index.ts:2]: Not quite sure why there's a `"` and a `'` - do you have prettier running? This reminds me, we should add a Prettier pre-commit hook 😎

**Review by colbr — CHANGES_REQUESTED** [review]
Looks great! These will need tests, but since they're not going into a production app for a while I'll let them through.

Only required changes are the `px` -> `rem` ones, and running Prettier. Nice work 🚀

**Comment by ashmidgley** [comment]
Cheers for the review @colbr! Must have let a rogue `npx prettier --write .` slip in there. Should be all sorted now!

**Review by colbr — APPROVED** [review]

**Pull request merged**